### PR TITLE
removed old HSR report

### PR DESCRIPTION
**Pull request by lwyszomi**
@sravfeyn

### CODE REVIEW

File: custom/opm/reports.py [hunk 1]
```diff
@@ -34,23 +34,20 @@
 from corehq.apps.es import cases as case_es, filters as es_filters
 from corehq.apps.reports.cache import request_cache
 from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
-from corehq.apps.reports.filters.dates import DatespanFilter
 from corehq.apps.reports.filters.select import MonthFilter, YearFilter
 from corehq.apps.reports.generic import ElasticTabularReport, GetParamsMixin
-from corehq.apps.reports.sqlreport import DatabaseColumn, SqlData, AggregateColumn, DataFormatter, DictDataFormat
-from corehq.apps.reports.standard import CustomProjectReport, MonthYearMixin, DatespanMixin
+from corehq.apps.reports.sqlreport import DatabaseColumn, SqlData, AggregateColumn
```
Comment (dimagimon) [inline custom/opm/reports.py]: F401 'AggregateColumn' imported but unused

**Comment by czue** [comment]
nice! :boom: :boom:

**Comment by sravfeyn** [comment]
Yay! Nice!

**Comment by sravfeyn** [comment]
@lwyszomi Just to make sure, did you check that other reports share none of this code and work correctly?
Can you also remove any functions/classes/pillows that are used exclusively by old-HSR? I found couple of such pieces like, `case_calcs.py`, `VhndAvailabilityFluff`and `VhndAvailabilityFluffPillow`. Let me know if you want to do that in a separate PR, I can merge this then.

**Comment by czue** [comment]
(I think the CMR still uses the vhnd stuff)

**Comment by sravfeyn** [comment]
Looks like CMR is using `opm.reports.SharedDataProvider` which uses raw vhnd forms and not the `VhndAvailabilityFluff`. But yeah, need to be careful to check all the dependencies while removing code.

**Comment by lwyszomi** [comment]
@sravfeyn 
I will update PR tomorrow.

**Comment by lwyszomi** [comment]
@sravfeyn 
I updated the PR, I checked and other reports works fine.

**Comment by czue** [comment]
+7 −793 :boom:

**Pull request merged**